### DISCUSSION TITLE
SQLSTATE[HY000]: General error: 1364 Field 'title' doesn't have a default value

### DISCUSSION THREAD

**Discussion by safarovitch** [spatie/laravel-translatable#330] [Q&A]
I am getting error when trying to create record using Model::create() method.

I used this package several times before and newer go this error but now it is throwing `SQLSTATE[HY000]: General error: 1364 Field 'title' doesn't have a default value`

The title field is json and it presents in `public $translatable` and it does not present in `protected $fillable`

**Comment by freekmurze** [comment]
All the test of this package indicate that it is functioning correctly. Could you PR a failing test (and possibly a fix)?

**Reply by safarovitch** [reply]
Nevermind!

This was my mistake. All the attributes that are in $translatable variable should also present in $fillable.
I mistakenly removed the translatable attributes from $fillable array which caused not saving data at all.

F.I.Y. The reason I removed attributes from $fillable was that I experienced anomaly when updating data. When field presents in $fillable it is being saved as translatable when first created but is being updated as normal field not translatable field. I guess that was some kind of conflict with other parts of my code.
